tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -865,7 +882,7 @@ class TrtGraphConverterV2(object):
   1. FP32/FP16 precision
 
      ```python
-     params = DEFAULT_TRT_CONVERSION_PARAMS._replace(
+     params = tf.experimental.tensorrt.ConversionParams(
```

Comment:
this module doesn't "import tensorflow as tf" so it can't use tf.experimental as a symbol (I don't think this code even runs?)